src/services/BrandingService.js

```diff
@@ -0,0 +1,75 @@
+import Service from '../abc/Service';
+
+export default class BrandingService extends Service {
+
+	/*
+	Branding service expects `header-logo-full.svg` and `header-logo-minimized.svg` SVG images
+	in `/public/media/logo/` directory:
+
+	* `header-logo-full.svg` dimensions: 120 x 30 pixels
+	* `header-logo-minimized.svg` dimensions: 30 x 30 pixels
+	*/
+
+	constructor( app, serviceName="BrandingService" ){
+		super(app, serviceName)
+		this.state = [];
+		this.App = app;
+
+		this.brandImage = undefined;
+		this.defaultBrandImage = undefined;
+		this.sidebarLogo = undefined
+	}
+
+	initialize() {
+		// which of the if statements is 'more' true?..
+		// if (this.App.Services.ConfigService && this.App.Modules.some(obj => obj.Name == "ASABConfigModule")) {
+		if (this.App.Services.ConfigService) {
+			this.brandImage = this.App.Config.get('brandImage');
+			this.defaultBrandImage = this.App.Config.get('defaultBrandImage');
+			this.sidebarLogo = this.App.Config.get('sidebarLogo');
+		}
+	}
+
+	determineSources(imgObject, theme) {
```

Comment:
@petrKavulok Since `determineSources` is a inner function (shouldnt be propagated out of the service), its always good to have `_` underscore before the name of the method to indicate, that it can be used only within the service and nowhere else.

```
_determineSources(...)
```